lib/components/buttons/buttonContainer.dart

```diff
@@ -177,6 +197,11 @@ class _ButtonContainer extends State<ButtonContainer> {
           text: 'Hey <3',
           textStyle: TextStyle(fontSize: 20, color: Colors.red),
           onPressed: () => this.onPressed(vm));
+      case '1.3':
```

Comment:
not sure if this is the best way of enabling sound on the button.
onPressed function have the vm where it can get achivement however like this we can keep the all the achivements in the set on top of the widget

Reply:
For now i think it's the best solution, we will just have to sent 'true' in every button onPressed that should have sound until we figure out a better way 👍